lib/course_planner/tasks/task.ex

```diff
@@ -30,6 +30,8 @@ defmodule CoursePlanner.Tasks.Task do
     |> validate_required(@required_params)
     |> validate_datetime()
     |> validate_expiration()
+    |> convert_timezone(:start_time)
```

Comment:
why the input time by the user needs convert ? cause user always put her time according to her own local timezone

Reply:
Because we want to store it in UTC
Say the user inserts a 14:00 in her timezone and stores it naively (not converting)
If we compare anywhere to a time generated by say `Timex.now`, it's gonna be "off".